api/src/controllers/cle/referent.js

```diff
@@ -0,0 +1,81 @@
+const express = require("express");
+const passport = require("passport");
+const router = express.Router();
+const Joi = require("joi");
+const crypto = require("crypto");
+const { capture } = require("../../sentry");
+const { SUB_ROLES, ROLES, SENDINBLUE_TEMPLATES, canInviteCoordinateur } = require("snu-lib");
+const { ERRORS, inSevenDays } = require("../../utils");
+const { sendTemplate } = require("../../sendinblue");
+const config = require("../../config");
+const EtablissementModel = require("../../models/cle/etablissement");
+const ReferentModel = require("../../models/referent");
```

Comment:
Je split juste l'import des libs des imports locaux

```suggestion
const { SUB_ROLES, ROLES, SENDINBLUE_TEMPLATES, canInviteCoordinateur } = require("snu-lib");

const { capture } = require("../../sentry");
const { ERRORS, inSevenDays } = require("../../utils");
const { sendTemplate } = require("../../sendinblue");
const config = require("../../config");
const EtablissementModel = require("../../models/cle/etablissement");
const ReferentModel = require("../../models/referent");
```